src/socket.cpp

```diff
@@ -28,6 +28,11 @@
 /* Implementation *************************************************************/
 void CSocket::Init ( const quint16 iPortNumber, const quint16 iQosNumber, const QString& strServerBindIP )
 {
+    // first store parameters, in case reinit is required (mostly for iOS)
+    iPortNumber_     = iPortNumber;
```

Comment:
This style isn't used elsewhere.  Please don't introduce new coding styles.  You don't need to do this, either:
```
this.varname = varname;
```
works.

Reply:
ok thanks!